rai/dicom/uid.py

```diff
@@ -21,12 +21,12 @@
 # valid UIDs (http://www.medicalconnections.co.uk/FreeUID.html)
 # Their service was used to obtain the following root UID for
 # Radiotherapy AI:
-RAI_ROOT_UID = "1.2.826.0.1.3680043.10.756."
+_RAI_ROOT_UID = "1.2.826.0.1.3680043.10.756."
 
-RAI_INTERNAL_ROOT_UID = f"{RAI_ROOT_UID}0."
-RAI_CLIENT_ROOT_UID = f"{RAI_ROOT_UID}1."
+_RAI_INTERNAL_ROOT_UID = f"{_RAI_ROOT_UID}0."
 
-PRODUCT_ID = "000000"
+_PRODUCT_ID = "000000"
 
-RAI_IMPLEMENTATION_CLASS_UID = f"{RAI_INTERNAL_ROOT_UID}{PRODUCT_ID}.{__version__}"
+RAI_CLIENT_ROOT_UID = f"{_RAI_ROOT_UID}1."
+RAI_IMPLEMENTATION_CLASS_UID = f"{_RAI_INTERNAL_ROOT_UID}{_PRODUCT_ID}.{__version__}"
```

Comment:
I believe that having .00 in a UID is not allowed.
_RAI_INTERNAL_ROOT_UID ends with a . (the 0 is ok there).
But if your product ID starts with 00, that's a problem.
Not that too many systems will complain about .00, but failing DICOM conformance is something to avoid.

Reply:
Good call, thanks @sjswerdloff, I didn't realise that. I will just change the product ID to 1, and then increment from there for other products.

Reply:
Fixed in [f681979](https://github.com/RadiotherapyAI/rai/pull/82/commits/f68197954350a7a497759ad2046b56e87a23ad7a)